src/resource_api/schema.py

```diff
@@ -190,8 +190,21 @@ def get_schema(self):
 
 
 class IndexableField(BaseSimpleField):
+    """
+    Base class for most of the primitive fields.
+
+    choices (list|tuple = None)
+        Predefined list of possible values for the field. ValidationError will be raised if field value doesn't
+        match any of the defined values.
+    choice_labels (list|tuple = None)
```

Comment:
Then again, enum name as a label may be quite limiting. These values could be extended with extra label field, but meh..
Using OrderedDict would solve ordering but allow e.g. spaces in the label.